### PR TITLE
Log widget is a default active widget

### PR DESCRIPTION
**Pull request by jacek-izykowski**

### CODE REVIEW

**Comment by mkow** [comment]
This doesn't work when starting Veles with command line arguments (files to open). In such case the last opened file should be active, but currently all tabs are inactive.

**Comment by jacek-izykowski** [comment]
@mkow Fixed.

File: src/ui/main.cc [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #include <QSurfaceFormat>
 #include <QTranslator>
 #include <QHostAddress>
-
+#include <iostream>
```
Comment (altran01) [inline src/ui/main.cc]: This is already included above

File: include/ui/veles_mainwindow.h [hunk 2]
```diff
@@ -71,7 +71,7 @@ class VelesMainWindow : public MainWindowWithDetachableDockWidgets {
   void createMenus();
   void createDb();
   void createFileBlob(QString);
-  void createHexEditTab(QString, dbif::ObjectHandle);
+  DockWidget* createHexEditTab(QString, dbif::ObjectHandle);
```
Comment (altran01) [inline include/ui/veles_mainwindow.h]: I'm missing how is this supposed to work. I can only find 2 places there createHexEditTab is used, and in both of them nothing is done with return value.

File: src/ui/main.cc (hunk 1 above)
Comment (jacek-izykowski) [inline src/ui/main.cc]: Fixed.

File: include/ui/veles_mainwindow.h (hunk 2 above)
Comment (jacek-izykowski) [inline include/ui/veles_mainwindow.h]: Unneeded code removed.

**Pull request merged**